.gitignore

```diff
@@ -0,0 +1,37 @@
+# Built application files
+/*/build/
```

Comment:
I think you need to remove this /*/ since you are committing the build files, causing lots of files to be merged. This can also cause merge conflict.

Also, remove these build files in your next commit so to reflect in the master when we merge your branch.